vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/validation/BaseValidations.java

```diff
@@ -14,48 +14,45 @@
  * limitations under the License.
  */
 
-package org.vividus.bdd.steps.ui.web.validation;
+package org.vividus.bdd.steps.ui.validation;
 
 import static org.hamcrest.Matchers.greaterThan;
 import static org.vividus.ui.validation.matcher.WebElementMatchers.elementNumber;
 
-import java.net.URI;
 import java.util.List;
 import java.util.Optional;
 
 import org.hamcrest.Matcher;
 import org.hamcrest.Matchers;
 import org.openqa.selenium.SearchContext;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.WrapsElement;
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.vividus.bdd.steps.ComparisonRule;
 import org.vividus.selenium.IWebDriverProvider;
+import org.vividus.ui.IState;
```

Comment:
Is this state valid for mobile apps?

Reply:
yes, basic verifications like is disabled, selected etc are applicable for mobiles